<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Actor</class>
 <widget class="QWidget" name="Actor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>799</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1187</width>
    <height>799</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rating">
         <property name="text">
          <string>Overall Rating</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="picture">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Picture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="Information">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="biography">
        <attribute name="title">
         <string>Biography</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="textBiography">
             <property name="minimumSize">
              <size>
               <width>1143</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="movies">
        <attribute name="title">
         <string>Movies</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="tableMovies"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="awards">
        <attribute name="title">
         <string>Awards</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QTableView" name="tableAwards"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="genres">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Genres</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableView" name="tableGenres"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
